db/re/item_db.conf

```diff
@@ -145960,6 +145960,27 @@ item_db: (
 
 //== New Etc Items =========================================
 {
+	Id: 22508
+	AegisName: "Para_Team_Mark_"
+	Name: "Eden Group Mark"
+	Type: 11
+	Buy: 0
+	Delay: 1200000
+	Trade: {
+		nodrop: true
+		notrade: true
+		noselltonpc: true
+		nocart: true
+		nostorage: true
+		nogstorage: true
+		nomail: true
+		noauction: true
+	}
+	Script: <"
```

Comment:
please add space after comma, and if the script is only one line like it is here, do it like this:
Script: <" unitskilluseid getcharid(3), "AL_TELEPORT" ,3; ">